--- libgtkpp-0.2.4/src/Makefile.am.orig	2003-05-15 05:01:24.000000000 +0200
+++ libgtkpp-0.2.4/src/Makefile.am	2004-02-22 02:29:36.913065536 +0100
@@ -1,4 +1,4 @@
-CXXFLAGS += -Wall
+AM_CXXFLAGS = -Wall
 
 libgtkppincludedir = $(includedir)/gtkpp
 
@@ -11,7 +11,7 @@
 #$(GTKHTML_INCLUDEDIR)
 
 libgtkpp_la_LDFLAGS = -release 0.2.4
-
+libgtkpp_la_LIBADD = $(GNOMEUI_LIBS) -lgdk_pixbuf
 libgtkpp_la_SOURCES = \
 about.cpp \
 arrow.cpp \
